ansible/roles/stack-sunbird/templates/ml-core-service.env

```diff
@@ -81,4 +81,16 @@ ELASTICSEARCH_ENTITIES_INDEX={{ml_core_elasticsearch_user_extension_index_type |
 USER_SERVICE_URL={{ml_core_user_service_URL | default("http://learner-service:9000")}}
 
 ## portal url of env
-APP_PORTAL_BASE_URL={{ proto }}://{{ domain_name }}
\ No newline at end of file
+APP_PORTAL_BASE_URL={{ proto }}://{{ domain_name }}
+
+# Kafka enable or disable communication flag
+KAFKA_COMMUNICATIONS_ON_OFF={{ml_core_kafka_communications_on_off | default("ON")}}
+
+# IP address of kafka server with port without HTTP
+KAFKA_URL={{ml_core_kafka_url | default(groups['processing-cluster-kafka'][0]+':9092')}}
+
+# Kafka consumer group for ML Core Service
+KAFKA_GROUP_ID={{ml_core_kafka_group_id | default("ml-core-service")}}
+
+#Kafka topic name for pushing programUsers
+PROGRAM_USERS_JOINED_TOPIC={{ml_core_submission_topic | default (env_name+".ml.programUsers.raw")}}
```

Comment:
@VISHNUDAS-tunerlabs why is the env override variable called - ml_core_submission_topic, why the word submission?

Reply:
@aks30 renamed the env variable. default value changed to - env_name+".programuser.info" - because data team told to keep it as ${job.env}".programuser.info.